### PR TITLE
Ci setup

### PR DESCRIPTION
**Pull request by somexpert**
Adding tox and Travis CI to the app so we can make sure new commits don't break things. Should fix issue #19.

### CODE REVIEW

File: major_event_log/models.py [hunk 1]
```diff
@@ -28,8 +28,8 @@ class Event(models.Model):
     entry_created = models.DateTimeField(auto_now_add=True)
     entry_modified = models.DateTimeField(auto_now=True)
     # The contact_name attribute will be used as the reporting agent.
-    contact_name = models.CharField(max_length=100, help_text=
-                                    "Name will appear as the Reporting Agent")
+    contact_name = models.CharField(max_length=100,
```
Comment (damonkelley) [inline major_event_log/models.py:31]: :100:

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py{27,34}-django18, flake8
+[flake8]
```
Comment (damonkelley) [inline tox.ini:4]: @somexpert can you add whitespace between the ini headers?

**Comment by somexpert** [comment]
@ldko, @damonkelley, what do you guys think? This good enough for this little branch? Just wanted to have CI setup for later branches.

**Comment by damonkelley** [comment]
Works for me :+1:

**Comment by ldko** [comment]
:+1:

**Pull request merged**